File: Java/Tests/jdi-uitest-tutorialtests/src/main/java/com/epam/jdi/uitests/testing/career/page_objects/entities/Attendee.java
package com.epam.jdi.uitests.testing.career.page_objects.entities;

/**
 * Created by Roman_Iovlev on 10/22/2015.
 */
public class Attendee {
    public JobSearchFilter filter = new JobSearchFilter();

    public String name = "Roman";
    public String lastName = "Iovlev";
    public String email = "roman_iovlev@epam.com";
    public String country;
    public String city = "Saint-Petersburg";
    public String cv; // = "jdi-uitest-tutorialtests\\src\\test\\resources\\cv.txt";
    public String comment = "I WANT TO WORK IN EPAM!!!";

    public Attendee() { }
    public Attendee(String name) {
        this.name = name;
    }
    @Override
    public String toString() {
        return name + " " + lastName;
    }
}

File: C# .Net/JDI UI Framework/Tests/TutorialTests/Career/PageObjects/Entities/Attendee.cs
namespace Epam.Tests.TutorialTests.Career.PageObjects
{
    public class Attendee
    {
        public JobSearchFilter filter = new JobSearchFilter();

        public string name = "Roman";
        public string lastName = "Iovlev";
        public string email = "roman_iovlev@epam.com";
        public string country;
        public string city = "Saint-Petersburg";
        public string cv;// = "jdi-uitest-tutorialtests\\src\\test\\resources\\cv.txt";
        public string comment = "I WANT TO WORK IN EPAM!!!";

        public Attendee() { }
        public Attendee(string name)
        {
            this.name = name;
        }
        
        public override string ToString()
        {
            return name + " " + lastName;
        }

    }
}
